dashboard/pages/fragment_graph.tsx

```diff
@@ -194,28 +198,32 @@ interface EmbeddedBackPressureInfo {
 
 export default function Streaming() {
   const { response: relationList } = useFetch(getStreamingJobs)
-  const { response: fragmentList } = useFetch(getFragments)
+  const { response: relationIdInfos } = useFetch(getRelationIdInfos)
 
   const [relationId, setRelationId] = useQueryState("id", parseAsInteger)
   const [selectedFragmentId, setSelectedFragmentId] = useState<number>()
+  const [tableFragments, setTableFragments] = useState<TableFragments>()
 
   const toast = useErrorToast()
 
+  useEffect(() => {
+    if (relationId) {
+      getFragmentsByJobId(relationId).then((tf) => {
```

Comment:
Shall we clean `tableFragments` first before fetching new fragments?

Reply:
Sorry I don't quite get it, we should we clean `TableFragments` first?
```
    if (relationId) {
      getFragmentsByJobId(relationId).then((tf) => {
        setTableFragments(tf)
      })
    }
```

Will fetching, then overriding previous table fragment have any issue?

Reply:
`getFragmentsByJobId` is asynchronous. If the network is slow, users may still see the old fragment graph for a while after selecting a new job. I'm just wondering if this can be improved to be less confusing.

Reply:
I got it. I will set it to `undefined first then`.